src/commands/show.ts

```diff
@@ -68,7 +68,7 @@ async function fetchProblemLanguage(): Promise<string | undefined> {
     const language: string | undefined = defaultLanguage || await vscode.window.showQuickPick(languages, { placeHolder: "Select the language you want to use" });
```

Comment:
We can also add `ignoreFocusOut: true` into the `showQuickPick()`'s option to avoid user close the drop list by mistake